spec/PrettyXml/FormatterSpec.php

```diff
@@ -75,11 +75,11 @@ function it_should_indent_double_nested_elements()
 
     function it_should_indent_a_nested_element_with_an_attribute()
     {
-        $this->format('<?xml version="1.0" encoding="UTF-8"?><foo><bar a="b">Baz</bar></foo>')
+        $this->format('<?xml version="1.0" encoding="UTF-8"?><foo a="b"><bar c="d">Baz</bar></foo>')
```

Comment:
Please would you add this as a separate spec to ensure that the original behaviour stays?